Cargo.toml

```diff
@@ -11,15 +11,16 @@ edition = "2021"
 [dependencies]
 clap = { version = "4.5.3", features = ["derive"] }
 clap_derive = "4.5.3"
-rust-htslib = "0.47.0"
+rust-htslib = "0.48.0"
 bio = "2.0.3"
 anyhow = "1.0.93"
-varlociraptor = "8.4.10"
+varlociraptor = { git = "https://github.com/fxwiegand/varlociraptor.git", branch = "patch-6" }
```

Comment:
This is not possible atm because varlociraptor uses a different strum version than rust bio and that causes issues. So I had to update the strum dep for varlociraptor on a branch and have to wait until its merged.